tools/build_variables.bzl

```diff
@@ -199,6 +199,7 @@ core_sources_full = [
     "torch/csrc/jit/passes/utils/subgraph_utils.cpp",
     "torch/csrc/jit/passes/xnnpack_rewrite.cpp",
     "torch/csrc/jit/passes/vulkan_rewrite.cpp",
+    "torch/csrc/jit/passes/quantization/qat_combine_conv_bn.cpp",
```

Comment:
nit: suggest to use `qat_fold_conv_bn` to be aligned with the inference version

Reply:
we aren't actually folding the bn in this pass though, this pass is fundamentally doing a different thing compared to the `fold_conv_bn` pass